lib/helper/Puppeteer.js

```diff
@@ -14,6 +14,74 @@ const ElementNotFound = require('./errors/ElementNotFound');
 
 const puppeteer = requireg('puppeteer');
 
+/**
+ * Class to handle the interaction with the Dialog (Popup) Class from Puppeteer
+ */
+class PopupStore {
```

Comment:
Please move this to a separate file. Maybe to `lib/helper/extra/Popup.js` or smth like that